From b94eff3461e4fb99cd761d6858e8ed94cb6c5412 Mon Sep 17 00:00:00 2001
From: Wang Mingyu <wangmy@cn.fujitsu.com>
Date: Tue, 8 Dec 2020 11:58:06 +0900
Subject: [PATCH] 0001-Modify-parameter-of-function-sctp_gensio_alloc.patch

parameter type of declearation is const, definition should also be
const.

Signed-off-by: Wang Mingyu <wangmy@cn.fujitsu.com>
---
 lib/gensio_sctp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/gensio_sctp.c b/lib/gensio_sctp.c
index bd9437f..4278ff8 100644
--- a/lib/gensio_sctp.c
+++ b/lib/gensio_sctp.c
@@ -1092,7 +1092,7 @@ str_to_sctp_gensio_accepter(const char *str, const char * const args[],
 #else
 
 int
-sctp_gensio_alloc(struct gensio_addr *iai, const char * const args[],
+sctp_gensio_alloc(const struct gensio_addr *iai, const char * const args[],
 		  struct gensio_os_funcs *o,
 		  gensio_event cb, void *user_data,
 		  struct gensio **new_gensio)
-- 
2.25.1

